ext/pdo_sqlite/tests/subclasses/pdo_sqlite_createaggregate.phpt

```diff
@@ -0,0 +1,32 @@
+--TEST--
+PDO_sqlite: Testing createAggregate()
+--EXTENSIONS--
+pdo_sqlite
+--FILE--
+<?php
+
+// This test was copied from the pdo_sqlite test for sqliteCreateAggregate
+$db = new PdoSqlite('sqlite::memory:');
+
+$db->query('CREATE TABLE IF NOT EXISTS foobar (id INT AUTO INCREMENT, name TEXT)');
+
+$db->query('INSERT INTO foobar VALUES (NULL, "PHP")');
+$db->query('INSERT INTO foobar VALUES (NULL, "PHP6")');
+
+$db->createAggregate('testing', function(&$a, $b) { $a .= $b; return $a; }, function(&$v) { return $v; });
+
+
+foreach ($db->query('SELECT testing(name) FROM foobar') as $row) {
+    var_dump($row);
+}
+
+$db->query('DROP TABLE foobar');
+
```

Comment:
Should be in a CLEAN section, also remove the IF NOT EXISTS in the the table creation query

Reply:
We don't even need a clean section since this is an in-memory database :)

Reply:
Remove the Drop query then?